### PR TITLE
feat: more explicitly request information in helpdesk message

### PR DESCRIPTION
**Pull request by bo0tzz**

### CODE REVIEW

File: src/discord/help-desk.ts [hunk 1]
```diff
@@ -330,6 +330,20 @@ ${tasks.map((task, index) => `${index + 1}. ${icon(index)} ${task}`).join('\n')}
...
+In order to be able to effectively help you, we need you to provide clear information to show what the problem is. The exact details needed vary per case, but here is a list of things to consider:
+- Your docker-compose.yml and .env files.
+- Logs from all the containers.
+- The status of the containers (\`docker ps -a\`).
+- Any recent changes you've made to Immich or your system.
+- Details about your system (both software/OS and hardware).
+- Details about your storage (filesystems, type of disks, output of commands like \`fdisk -l\` and \`df -h\`).
+- Any other information that you think might be relevant.
+
+Please paste files and logs with proper code formatting (three backticks \`\`\`), and especially avoid blurry screenshots.
```
Comment (bo0tzz) [inline src/discord/help-desk.ts:344]: Will these backticks format right?

File: src/discord/help-desk.ts [hunk 2]
```diff
@@ -330,6 +330,20 @@ ${tasks.map((task, index) => `${index + 1}. ${icon(index)} ${task}`).join('\n')}
...
+We're not in your computer, so without the right information we can't work out what the problem is. Help us help you ;)
```
Comment (bo0tzz) [inline src/discord/help-desk.ts:345]: Is this too snarky? :P

File: src/discord/help-desk.ts [hunk 3]
```diff
@@ -330,6 +330,20 @@ ${tasks.map((task, index) => `${index + 1}. ${icon(index)} ${task}`).join('\n')}
...
+## Information
+
+In order to be able to effectively help you, we need you to provide clear information to show what the problem is. The exact details needed vary per case, but here is a list of things to consider:
+- Your docker-compose.yml and .env files.
+- Logs from all the containers.
+- The status of the containers (\`docker ps -a\`).
+- Any recent changes you've made to Immich or your system.
+- Details about your system (both software/OS and hardware).
+- Details about your storage (filesystems, type of disks, output of commands like \`fdisk -l\` and \`df -h\`).
+- Any other information that you think might be relevant.
```
Comment (bo0tzz) [inline src/discord/help-desk.ts:342]: Suggestions for more entries are welcome

File: src/discord/help-desk.ts (hunk 1 above)
Comment (jrasm91) [reply src/discord/help-desk.ts:344]: I think this will be three back ticks which will start a code block.

Comment (jrasm91) [reply src/discord/help-desk.ts:344]: Maybe three like you have it, then a single one, and then another three.

File: src/discord/help-desk.ts (hunk 2 above)
Comment (bo0tzz) [reply src/discord/help-desk.ts:345]:
```suggestion
We're not in your computer, without the right information we can't work out what the problem is. Help us help you ;)
```
You mean this?

Comment (bo0tzz) [reply src/discord/help-desk.ts:345]: Or like this?

```suggestion
Without the right information we can't work out what the problem is. Help us help you ;)
```

Comment (danieldietzler) [reply src/discord/help-desk.ts:345]: I think Jason means the second one (thus 'Without' being capitalized), and I do also agree with that. I like the original text, but I think removing that is more appropriate. (same vein as /rtfm)

**Review by danieldietzler — APPROVED** [review]
This looks really good to me!

**Pull request merged**